database-portfolio/src/main/webapp/script.js

```diff
@@ -13,42 +13,77 @@
 // limitations under the License.
 //
 
+async function getUserLoginStatus() {
+  const url = '/login';
+  let response = await fetch(url);
+  let text = await response.text();
+  return text;
+}
+
 /** Loads and displays comments according to a limit */
-function loadCommentSection(commentLimit) {
-    // clear previous elements
-    var currentDOM = document.getElementsByClassName('indiv-comments');
-    if (currentDOM != null) {
-        console.log(currentDOM);
-        Array.from(currentDOM).forEach(function (element) {
+async function loadCommentSection(commentLimit) {
+    let userLoginInfo = await getUserLoginStatus();
+    let isLoggedIn = "Y" === userLoginInfo[0];
+
+    // clear previous elements to refresh login status
+    var loginDOM = document.getElementsByClassName('login');
+    if (loginDOM != null) {
+        Array.from(loginDOM).forEach(function (element) {
+            element.remove();
+        })
+    }
+
+    // clear previous elements to refresh comments
+    var commentDOM = document.getElementsByClassName('indiv-comments');
+    if (commentDOM != null) {
+        Array.from(commentDOM).forEach(function (element) {
             element.remove();
         })
     }
 
-    commentURL = `/list-comments?limit=${commentLimit}`;
-    fetch(commentURL)
-      .then(response => response.json())
-      .then((comments) => {
-      // Build the list of comments.
-      const commentLe = document.getElementById('post-content');
-      for (let comment of comments) {
-        commentLe.appendChild(createCommentElement(comment));
-      }
-    });
+    // split text stream by new line character, use REGEX to put in HTML
+    const words = userLoginInfo.split("\n");
+    for (let i = 1; i < words.length - 1; i++) {
+        const loginSection = document.getElementById('content');
+        const htmlTag = words[i].match(/^<(.*?)>/)[1];
+        const textContent = words[i].match(/>(.*)</)[1];
+        const loginElement = document.createElement(htmlTag);
+        loginElement.setAttribute("class", "login");
+        loginElement.innerHTML = textContent;
+        loginSection.appendChild(loginElement);
+    }
+
+    // if user is logged in, display comments
+    if (isLoggedIn) {
+      const userEmail = userLoginInfo.match(/\S*@\S*[com|edu|org|gov]/);
+      document.getElementById('comment').value = userEmail;
+      document.getElementById('post-content').style.display = "block";
+      commentURL = `/list-comments?limit=${commentLimit}`;
```

Comment:
Where is this defined? It appears to be a global variable. It should probably be const.